theano/gpuarray/opt.py

```diff
@@ -1884,6 +1886,14 @@ def typebuild(dtype, broadcastable, context_name=context_name):
 def local_gpu_maxandargmax(op, context_name, inputs, outputs):
     return GpuMaxAndArgmax(op.get_params(None))
 
+
+# solve
+@register_opt('fast_compile')
```

Comment:
Put a "if cusolver_available" (import it from gpuarray.linalg) so that the op_lifter is not applied if the optional package is missing.

@nouiz , @abergeron , do we prefer to have the op_lifter in gpuarray.opt or the op file itself?